.github/workflows/bicep_workflow_feature.yml

```diff
@@ -46,7 +46,6 @@ jobs:
       REMOTE_VNET_NAME_VPN: ${{ secrets.REMOTE_VNET_NAME_VPN }}
       VNET_ADDRESS_PREFIX: ${{secrets.VNET_ADDRESS_PREFIX}}
       UKEF_VPN_IPS: ${{ secrets.UKEF_VPN_IPS }}
-
```

Comment:
That was there to give a delineation between "infrastructure" values and application environment variables. I don't feel strongly about it. Either leave it as is or put a comment saying as such.